LLVMFloatToFixed/IndirectCallConversion.cpp

```diff
@@ -0,0 +1,96 @@
+#include "LLVMFloatToFixedPass.h"
+#include "TypeUtils.h"
+#include <llvm/ADT/SmallPtrSet.h>
+#include <llvm/IR/Constants.h>
+#include <llvm/IR/Function.h>
+#include <llvm/IR/InstIterator.h>
+#include <llvm/IR/Instructions.h>
+#include <llvm/IR/Intrinsics.h>
+#include <llvm/IR/Module.h>
+#include <llvm/Transforms/Utils/Cloning.h>
+#include <llvm/Transforms/Utils/ValueMapper.h>
+
+using namespace llvm;
+using namespace taffo;
+using namespace flttofix;
+
+using handler_function = void (FloatToFixed::*)(
+    llvm::CallInst *patchedDirectCall, llvm::Function *indirectFunction);
+
+/// Map to keep track of the handled indirect functions for the conversion.
+const std::map<const std::string, handler_function> indirectCallFunctions = {
+    {"__kmpc_fork_call", &FloatToFixed::handleKmpcFork}};
+
+/// Retrieve the indirect calls converted into trampolines and re-use the
+/// original indirect functions.
+void FloatToFixed::convertIndirectCalls(llvm::Module &m) {
```

Comment:
```suggestion
void FloatToFixed::convertIndirectCalls(llvm::Module &m)
{
```
Style: opening braces on a newline for methods and functions in cpp files (not in headers). Fix elsewhere where applicable.